tezos_interop/crypto.rei

```diff
@@ -0,0 +1,43 @@
+open Helpers;
+open Mirage_crypto_ec;
+
+let blake2b_20_encoding: Data_encoding.t(BLAKE2B_20.t);
+module type S = {
```

Comment:
Any tip to avoid duplicating this module type?

Reply:
used an intf file